Docs for k8s Worker (#2416)
OctopusDeploy/docs: src/pages/docs/infrastructure/workers/autoscaling-workers/storage.md

---
layout: src/layouts/Default.astro
pubDate: 2024-04-29

> modDate: 2024-07-31

Reviewer: 2024-08-20?

title: Storage
description: How to configure storage for a Kubernetes worker
navOrder: 10
---

> The Kubernetes worker requires a common filesystem to share packages with its spawned operation pods. This filesystem

Reviewer 2: Captalise worker here?

> The Kubernetes worker requires a common filesystem to share packages with its spawned operation pods. This filesystem is
> stores binary packages received from the Octopus Server, and are used by the operation being executed.

Reviewer 3: ```suggestion
The Kubernetes worker requires a common filesystem to share packages with its spawned operation pods. This filesystem
stores binary packages received from the Octopus Server, which are used by the operation being executed.
```

> The Kubernetes worker's storage setup and constraints are identical to the [Kubernetes Agent storage](docs/kubernetes/targets/kubernetes-agent/storage.md).

Reviewer 2: This link doesn't go anywhere in the deployed version of the docs

Reviewer 2: Captalise worker here?
